### ISSUE TITLE
Texture coordinates of Image1D sampler misinterpreted

### ISSUE THREAD

**Issue by UX3D-haertl** [NVIDIA/VisRTX#45]
Button corners are always 0.
Top corners are 2.
![Image1D_None_mirrorRepeat_None_None_color](https://github.com/NVIDIA/VisRTX/assets/50208655/3252a49f-7b23-4799-9c22-a1c4c103970a)
Top corners are 1 (inTransform scaled the texture coordinate from 2 to 1).
![Image1D_None_None_ 0 5, 0, 0, 0 ,  0, 0 5, 0, 0 ,  0, 0, 0 5, 0 ,  0, 0, 0, 1 _None_color](https://github.com/NVIDIA/VisRTX/assets/50208655/d901f019-0af3-4f89-94ce-4d35f68c7248)
Top corners are 2. outTransform extracted only the red channel.
![Image1D_None_None_None_ 1 0, 0, 0, 0 ,  0, 0, 0, 0 ,  0, 0, 0, 0 ,  0, 0, 0, 1 _color](https://github.com/NVIDIA/VisRTX/assets/50208655/214f5f31-af4f-488b-b3cc-6a6915c14371)

**Comment by UX3D-haertl** [comment]
It seems like multi-channel colors are not represented correctly.
E.g. (0.5, 0.5, 0.5) or all colors with higher values are displayed as white. Therefore, the Image1D sampler works correctly, but the colors are wrong.
I also tested this by setting these values directly in a matte material

**Comment by jeffamstutz** [comment]
In looking at things closer, this may be that VisRTX was using a non-zero default for `ambientRadiance` on the renderer. I've changed this now on `next_release`: could you try again and see if that helps? In local testing, that seems to be the culprit.

**Comment by UX3D-haertl** [comment]
Yes, since we also still used directional lights the scene was to bright.
I now only use ambientRadiance, but I'm not sure what is the correct value to get the actual input colors.
E.g. 1.0 is still too bright while 0.75 is too dim. Is this expected? What is a reasonable value?

Currently, visrtx does not build on Windows since a Windows header file is included which defines OPAQUE (which results in a conflict). I fixed this by undefining it in Material.h
